api/src/main/data/bt_language_names.xml

```diff
@@ -414,6 +458,61 @@ QUESTIONS?
 <symbol>MALAYALAM</symbol>
 </language>
 <language>
+<id>107</id>
+<name>Marathi</name>
+<doc>Marathi</doc>
+<iso639-3>mar</iso639-3>
+<iso639-1>mr</iso639-1>
+<script>Deva</script>
+<symbol>MARATHI</symbol>
+</language>
+<language>
+<id>103</id>
+<name>Mongolian</name>
+<doc>Mongolian</doc>
+<iso639-3>mon</iso639-3>
+<iso639-1>mn</iso639-1>
+<script>Mong</script>
+<symbol>MONGOLIAN</symbol>
+<aux_name0>Mongolian (macrolanguage)</aux_name0>
+</language>
+<language>
+<id>104</id>
+<name>Mongolian, Halh</name>
+<doc>Halh Mongolian</doc>
+<iso639-3>khk</iso639-3>
+<script>Mong</script>
+<symbol>HALH_MONGOLIAN</symbol>
+</language>
+<language>
+<id>105</id>
+<name>Mongolian, Peripheral</name>
+<doc>Peripheral Mongolian</doc>
+<iso639-3>mvf</iso639-3>
+<script>Mong</script>
+<symbol>PERIPHERAL_MONGOLIAN</symbol>
```

Comment:
Is any Basis product ever going to distinguish Halh Mongolian from Peripheral Mongolian? I suppose the answer is that we don’t know but that they might so we may as well add them now. If so, should we add the many individual languages of the macrolanguages Albanian, Arabic, Estonian, Kurdish, Latvian, Malagasy, and Pushto?